behat/symfony2-extension/2.1/post-install.txt

```diff
@@ -0,0 +1,11 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * <fg=blue>Init Behat</>:
+    1. Change to the project directory
+    2. Execute the <comment>vendor/bin/behat --init</> command;
```

Comment:
I don't think we need to document how to use behat when they installed an extension for behat (thus, the way to start using Behat may change, and what people should do to use behat is described in the Behat doc.
so I suggest removing this message.